content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/maximum-image-age-policy.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Limit the “Build Horizon” of Build Artifacts with the Maximum Image Age Policy"
+type: "article"
+description: "Understanding Identity and Access Management in Chainguard Enforce"
+date: 2023-10-04T15:22:20+01:00
+lastmod: 2023-10-04T15:22:20+01:00
+draft: false
+images: []
+tags: ["policy-controller", "Policies", "Enforce", "Product"]
+menu:
+  docs:
+    parent: "chainguard-enforce-kubernetes"
+weight: 100
```

Comment:
Can you increase the weight so it is lower in the page? I would look at what weight the other policy tutorials have